ecosystem/sep-0010.md

```diff
@@ -65,20 +65,23 @@ GET https://auth.example.com/?account=GCIBUCGPOHWMMMFPFTDWBSVHQRT4DIBJ7AD6BZJYDI
 
 #### Response
 
-On success the endpoint should return `200 OK` HTTP status code and a JSON object with a `transaction` field containing an XDR-encoded Stellar transaction with the following:
+On success the endpoint must return `200 OK` HTTP status code and a JSON object with these fields:
 
-* source account set to server's signing account
-* invalid sequence number (set to 0) so the transaction cannot be run on the Stellar network
-* time bounds: `{min: now(), max: now() + 300 }` (we recommend expiration of 5 minutes to give user time to sign transaction)
-* operations: `manage_data(source: client_account, key: '<anchor name> auth', value: random_nonce())`
-  * The value of key is not important, but can be the name of the anchor followed by `auth`. It can be at most 64 bytes.
-  * The value must be a 64 byte long base64 encoded cryptographic-quality random string
-* signature by the web service signing account
+* `transaction`: an XDR-encoded Stellar transaction with the following:
+  * source account set to server's signing account
+  * invalid sequence number (set to 0) so the transaction cannot be run on the Stellar network
+  * time bounds: `{min: now(), max: now() + 300 }` (we recommend expiration of 5 minutes to give user time to sign transaction)
+  * operations: `manage_data(source: client_account, key: '<anchor name> auth', value: random_nonce())`
+    * The value of key is not important, but can be the name of the anchor followed by `auth`. It can be at most 64 bytes.
+    * The value must be a 64 byte long base64 encoded cryptographic-quality random string
+  * signature by the web service signing account
+* `network_passphrase`: (optional but recommended) Stellar network passphrase used by the server. This allows the client to verify that it's using the correct passphrase when signing.
```

Comment:
Looks good to me. I'm a little bit worried that `network_passphrase` is optional. It requires code that handles this situation and if developer forgets about it it may result in null pointer exceptions or using empty passphrase.

Reply:
Agreed. I think this is a necessary evil to make the change backwards compatible.